<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addcar</class>
 <widget class="QWidget" name="addcar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление автомобиля в каталог</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="gbox_addcar">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>229</red>
           <green>228</green>
           <blue>228</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>242</red>
           <green>241</green>
           <blue>241</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>114</green>
           <blue>114</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>152</green>
           <blue>152</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>229</red>
           <green>228</green>
           <blue>228</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>242</red>
           <green>241</green>
           <blue>241</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>229</red>
           <green>228</green>
           <blue>228</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>242</red>
           <green>241</green>
           <blue>241</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>114</green>
           <blue>114</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>152</green>
           <blue>152</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>229</red>
           <green>228</green>
           <blue>228</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>242</red>
           <green>241</green>
           <blue>241</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>114</green>
           <blue>114</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>229</red>
           <green>228</green>
           <blue>228</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Light">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Midlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>242</red>
           <green>241</green>
           <blue>241</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>114</green>
           <blue>114</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Mid">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>153</red>
           <green>152</green>
           <blue>152</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>114</green>
           <blue>114</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="BrightText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>114</red>
           <green>114</green>
           <blue>114</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>229</red>
           <green>228</green>
           <blue>228</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>229</red>
           <green>228</green>
           <blue>228</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Shadow">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="AlternateBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>229</red>
           <green>228</green>
           <blue>228</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipBase">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>220</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ToolTipText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Заполните анкету автомобиля:</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>371</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_exit">
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>204</x>
        <y>30</y>
        <width>181</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLineEdit" name="edt_firm"/>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_model"/>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_year"/>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_power"/>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_eng_type">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>инжектор</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>турбодизель</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>дизель</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>бензин-турбина</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>битурбо</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>компрессор</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_body_type">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Седан</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Авант</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Джип</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Минивен</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Кабриолет</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Купе</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Хетч-бек</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_drive_type">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Передний</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Задний</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Полный</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_rudder_type">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Левый</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Правый</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_color"/>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_condition">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Отличное</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Хорошее</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Среднее</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Битый</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_cond">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Да</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нет</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_tv">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Да</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нет</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_navig">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Да</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нет</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_present">
         <property name="editable">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Да</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нет</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_outgo"/>
       </item>
       <item>
        <widget class="QLineEdit" name="edt_price"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Фирма автомобиля:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Модель автомобиля:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Год выпуска:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Мощность двигателя (л/c):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Тип двигателя:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Тип кузова:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Привод:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Руль:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Цвет:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Состояние автомобиля:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Кондиционер:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>TV:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Навигация:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Наличие:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Расход на 100 км (л):</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Цена:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
